index.html

```diff
@@ -87,15 +87,21 @@ <h2>Silver Sponsor{% if silver_sponsors[1] %}s{% endif %}</h2>
       {% endfor %}
     </ul>
     {% endif %}
-
+    -->
     <div style="text-align: right">
-      <a class="button primary" href="/sponsors/">
-        Sponsors
+      <a class="button primary" href="https://cfp.rustfest.eu/events/rustfest-rome-2018">
+        Submit to CFP
       </a>
+      <a class="button primary" href="https://ti.to/asquera-event-ug/rustfest-rome-2018/">
+        Supporter Tickets 
+      </a>
+      <!-- <a class="button primary" href="/sponsors/">
+        Sponsoring Information
+      </a> -->
```

Comment:
Any reason not to activate the sponsoring info while we are at it?

Reply:
@skade no reason. I was simple not sure if it was ok to activate it :)

Reply:
can i activate it?

Reply:
Yes, though the correct URL is: /sponsors/packages/

There's a couple of tiny things I want to fix, but I can do that later.